jobs/ci-kubernetes-e2e-gce-federation.env

```diff
@@ -8,7 +8,7 @@ KUBE_GCS_RELEASE_BUCKET=kubernetes-federation-release
 KUBE_GCS_DEV_RELEASE_BUCKET=kubernetes-federation-release
 KUBE_NODE_OS_DISTRIBUTION=debian
 
-GINKGO_TEST_ARGS=--ginkgo.focus=\[Feature:Federation\]
+GINKGO_TEST_ARGS=--ginkgo.focus=\[Feature:Federation\] --ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]
```

Comment:
we could just split up the Serial test cases and skip them for current CRI and GCI test jobs